atlp-rwanda/alpha-ec-fe
# #[187300100] [Seller auth & Two Factor Authentication for Sellers]

#### What does this PR do?

Implement Seller Auth & Two-Factor Authentication for Sellers

#### Description of Task to be completed?

This pull request introduces a two-factor authentication (2FA) feature specifically for sellers on our platform.

#### How should this be manually tested?

npm run dev 
and then do login and make sure that you are seller just to verify OTP.

#### What are the relevant pivotal tracker/Trello stories?

#187300100

#### Screenshots (if appropriate)
